### ISSUE TITLE
base-64 encoded images falsely being detected as poster comments

### ISSUE THREAD

**Issue by migstopheles** [SassDoc/sassdoc#331]
Example:

```scss
.my-icon {
    background: url(data:image/png;base64,blahblahblah////blahblah);
}
```

More than one of these per file causes `[WARNING] Error: You can't have more than one poster comment.` Docs are still being generated so this isn't a dealbreaker, but it's not desired behaviour.

**Comment by HugoGiraudel** [comment]
Interesting. Thanks for the report Mike.

**Comment by migstopheles** [comment]
Should also add that I'm on Ubuntu 14.04, SassDoc 1.10.12

**Comment by valeriangalliat** [comment]
The parser is just missing a `^` [here](https://github.com/FWeinb/CDocParser/blob/52d8632e7c36d1949f2f6bf6c46e2bcab26ab8e6/index.js#L44).

I'm making a PR to CDocParser.

**Comment by valeriangalliat** [comment]
@migstopheles It's fixed in CDocParser 0.12.2; you need to upgrade to [SassDoc 2.0](http://sassdoc.com/develop/) to get the fix.

**Comment by migstopheles** [comment]
:+1:
